Server/src/config/envs/local.ts

```diff
@@ -121,8 +121,8 @@ export function createLocalConfig() {
       },
     },
     logLevel: 2, // debug
-    maxCallLengthSeconds: 5,
-    maxCachingSeconds: 10,
+    maxCallLengthSeconds: 30,
+    maxCachingSeconds: 30,
```

Comment:
Are these changes temp to testing?

Reply:
I was thinking these be permanent change.

It seems useful to have longer times so that we have enough time to manually send a response.

But maybe we can have input from the team.

Reply:
Got it. Makes sense.